web/pdf_find_controller.js

```diff
@@ -43,47 +43,144 @@ const CHARACTERS_TO_NORMALIZE = {
   "\u00BE": "3/4", // Vulgar fraction three quarters
 };
 
+const diacriticsRegExp = /\p{Mn}+/gu;
+const escapeRegExp = /[.*+\-?^${}()|[\]\\]/g;
+const prepareNoDiacriticsRegExp = /(\p{Mn})|(\p{L})/gu;
+const whitespacesRegExp = /\s+/g;
+const notDiacriticFromEndRegExp = /([^\p{Mn}])\p{Mn}*$/u;
+const notDiacriticFromStartRegExp = /^\p{Mn}*([^\p{Mn}])/u;
```

Comment:
Since these are true constants, please use the upper-case format instead (e.g. `DIACRITICS_REG_EXP` and similar).